### DISCUSSION TITLE
Parallel execution of solver runs.

### DISCUSSION THREAD

**Discussion by jusschwitalla** [QUARK-framework/QUARK#51] [Ideas]
I would like to start some solver runs in parallel. The concrete situation in my case is that I want to run some quantum jobs via QLMaaS. The job submit in QLMaaS returns an asynchronous result object which can be used some time later to get access to the results. 

What I would need is support in QUARK for this asynchronous mode. Are there already thought into that direction?

Best wishes,
Jürgen

**Comment by philross** [comment]
Hi Jürgen,

there is no ongoing work in that area, but I think it would be a good idea to discuss.
While I think the default should stay non-parallel, since a parallel run could have an impact on the measurements, an option for the user to enable that multiple benchmark runs can be executed in parallel might be a good idea.

**Comment by Marvmann** [comment]
Let's dicuss possible ways of parallelization in our next Open Call on June 22. Until then we will think about options and how to implement them.
